funding/manager.go

```diff
@@ -1743,7 +1740,7 @@ func (f *Manager) handleFundingOpen(peer lnpeer.Peer,
 		HtlcPoint:             ourContribution.HtlcBasePoint.PubKey,
 		FirstCommitmentPoint:  ourContribution.FirstCommitmentPoint,
 		UpfrontShutdownScript: ourContribution.UpfrontShutdown,
-		ChannelType:           chanTypeFeatureBits,
+		ChannelType:           msg.ChannelType,
```

Comment:
I think this is more than just refactoring and is an actual bug fix as we'd previously passed a `nil` value here. Could you update the commit message to reflect that?